manifests/init.pp

```diff
@@ -58,6 +58,8 @@
   Stdlib::Filemode $log_mode                                 = $nginx::params::log_mode,
   Variant[String, Array[String]] $http_access_log            = "${log_dir}/${nginx::params::http_access_log_file}",
   Optional[String] $http_format_log                          = undef,
+  Variant[String, Array[String]] $stream_access_log          = "${log_dir}/${nginx::params::stream_access_log_file}",
```

Comment:
Can you please make the type more strict?
```suggestion
  Variant[Stdlib::Absolutepath, Array[Stdlib::Absolutepath]] $stream_access_log = "${log_dir}/${nginx::params::stream_access_log_file}",
```

Reply:
And Instead of hardcoding the data in the params.pp, we can directly set it in the init.pp
```suggestion
  Variant[Stdlib::Absolutepath, Array[Stdlib::Absolutepath]] $stream_access_log = "${log_dir}/stream-access.log",
```